packages/material-ui-system/src/propsToClassKey.js

```diff
@@ -8,6 +8,7 @@ function isEmpty(string) {
  * Generates string classKey based on the properties provided. It starts with the
  * variant if defined, and then it appends all other properties in alphabetical order.
  * @param {object} props - the properties for which the classKey should be created
+ * entropy
```

Comment:
This looks lost.

Reply:
I've updated the comment. I could still successfully build the docs